lib/improver/utilities/temporal.py

```diff
@@ -35,6 +35,7 @@
 from datetime import datetime
 from datetime import time as dt_time
 from datetime import timedelta
+from datetime import timezone
 from time import mktime
```

Comment:
Maybe this was the the case before the changes, but:
`from time import mktime`
not required?

Reply:
I've removed this import.